crates/core/src/table/partitions.rs

```diff
@@ -289,59 +267,6 @@ impl TryFrom<(&str, &HashMap<String, DataType>)> for HudiTablePartition {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use serde_json::json;
-
-    fn check_json_serialize(filter: PartitionFilter, expected_json: &str) {
-        assert_eq!(serde_json::to_value(filter).unwrap(), json!(expected_json))
-    }
-
-    #[test]
-    fn test_serialize_partition_filter() {
```

Comment:
@KnightChess why the serialization test was removed?

Reply:
@xushiyan arrow scalar not implement `Display`, and I found the scenes that are not used at present, so I remove `Serialize` trait. If necessary, I can add implementation

Reply:
@xushiyan thank you so much